lib/public/views/Tags/Tags.js

```diff
@@ -292,7 +333,9 @@ class TagModel extends Observable {
      * @returns {undefined}
      */
     clearEditor() {
-        this.text = '';
+        this._text = '';
+        this._mattermost = '',
+        this._email = '',
```

Comment:
As we change the state we should also notify about it, especially as one of the objects is a RemoteData one